beast/tools/tests/test_compare_spec_type.py

```diff
@@ -0,0 +1,106 @@
+from beast.tools import compare_spec_type
```

Comment:
Better to just import the specific function needed.  So this could be `from beast.tools.compare_spec_type import compare_spec_type` and then the function can be called with just `compare_spec_type` instead of `compare_spec_type.compare_spec_type`.

Reply:
I thought it was best coding practice to import the module rather than the function.  But I guess there's not really ambiguity in the name `compare_spec_type`!